src/svxlink/contrib/module_site_status/ModuleSiteStatus.conf

```diff
@@ -0,0 +1,81 @@
+[ModuleSiteStatus]
+NAME=SiteStatus
+PLUGIN_NAME=Tcl
+ID=123123123
+TIMEOUT=1
```

Comment:
TIMEOUT is not required so it can be left out since the module cannot be activated anyway

Reply:
This one?